### DISCUSSION TITLE
Shape of test sample is different in YT video and notebbok on https://www.learnpytorch.io/02_pytorch_classification/

### DISCUSSION THREAD

**Discussion by Durga-33** [mrdbourke/pytorch-deep-learning#254] [Q&A] [answered]
Video: 02. PyTorch Neural Network Classification
Timestamp: 9:54:07

Output in YT video 
**Length of test samples: 200 and Shape: torch.size ([200,2])**

<img width="372" alt="image" src="https://user-images.githubusercontent.com/122006734/211886416-47b5fdf5-fc40-4520-9a4f-e74839245e7e.png">

output in 
https://www.learnpytorch.io/02_pytorch_classification/
shows

Length of predictions: 200, Shape: torch.Size([200, 1])
**Length of test samples: 200, Shape: torch.Size([200])**
<img width="448" alt="image" src="https://user-images.githubusercontent.com/122006734/211886596-527ee4e9-fbec-411c-889c-1f4c3f3a7278.png">

Also why there is so much difference in predictions values given in YT and given in notebook

Please clarify

**Comment by mrdbourke** [comment, marked as answer]
Hi @Durga-33,

Good question(s)!

You're right about the shapes being different in the videos/notebooks.

I'd pay more attention to the notebooks, the videos have some mistakes in them but the notebooks are checked regularly to make sure they work.

I've just rerun the notebook you're talking about and it works for me.

If the random numbers are *very* different, make sure the seeds you're setting are the same as in the videos/notebooks.

Otherwise, small differences are to be expected depending on the computing platform you're using.

**Reply by Durga-33** [reply]
Okay Sir
Thank you
